### ISSUE TITLE
`DIRECTORY_SEPARATOR` issue with Modular running in Windows

### ISSUE THREAD

**Issue by Sophist-UK** [InterNACHI/modular#40]
```
InterNACHI\Modular\Exceptions\CannotFindModuleForPathException
Unable to determine module for 'C:\Laravel\app-modules/Developer/resources'
```

As you can see the `$path` contains a mixture of `\` and `/`. I believe that this then triggers an issue in the `extractModuleNameFromPath` function in `Support/ModuleRegistry.php`.

In essence developers of PHP code have a choice - they can either:

1. Ensure that all paths use the `DIRECTORY_SEPARATOR` every time; or
2. Work on the basis of using `/` all the time (which generally are converted by PHP to `\` on Windows) and just make sure to replace `\` with `/` whenever you get a real path given by the user or by PHP. A little internet research suggests that this is preferred / recommended because URLs use `/`.

It appears that in InterNachi Modular we have a mixed usage.

**Comment by Sophist-UK** [comment]
P.S. I should add that this is a new Laravel project. I installed Laravel and Modular and debugbar and created a couple of modules and went to test that the Laravel welcome page still displayed and got this error.

**Comment by inxilpro** [comment]
@Sophist-UK can you try the `dev-main` release and confirm that your PR has resolved the Windows path issues?

**Comment by Sophist-UK** [comment]
I can't test this as soon as my laptop is back running (awaiting new power brick).

**Comment by Sophist-UK** [comment]
I have just tested this and can confirm that it fixed the issue I was having above.

**Comment by Sophist-UK** [comment]
Fixed by #38
